docs/contributing.rst

```diff
@@ -460,17 +459,26 @@ needed for documentation and checks for any problems such as broken links or syn
 Solidity Language Design
 ========================
 
-If you want to get involved in the language design process and share your ideas, please join the `solidity-users forum <https://groups.google.com/g/solidity-users>`_,
-where existing properties of the language and proposals for new features can be discussed.
+To actively get involved in the language design process and share your ideas concerning the future of Solidity,
+please join the `Solidity forum <https://forum.soliditylang.org/>`_, where existing properties of the language
+and proposals for new features can be discussed.
 
-We regularly host language design discussion calls, in which selected topics, issues or feature implementations are debated in detail. The invitation
-to those calls is shared via the aforementioned forum. We are also sharing feedback surveys and other language design relevant content in this forum.
+In the ``language design`` category, you can propose and discuss new language features and their implementation in
+the early stages of ideation or modifications of existing features. The forum serves as the place to discuss the
+initial soundness of the ideas/proposals and to define them further. As soon as proposals get more tangible their
```

Comment:
```suggestion
initial soundness of the ideas/proposals and to define them further. As soon as proposals get more tangible, their
```

Reply:
This paragraph reads slightly complicated. Maybe the first line can be just:


> In the ``language design`` category, you can propose and discuss new language features or modifications of existing features.

followed by

> The forum serves as a place to discuss proposals and refine them.

Yeah, but this is just subjective, anyways.